index.rst

```diff
@@ -4,9 +4,93 @@ Night Planning Tool
 
 .. abstract::
 
-   This technote describes the design of an integrated night planning tool. This is a tools that allows our team to collect a group of Jira tickets that describes a set of tests, allows us to organize these into a coherent list of tests to execute, that can later be loaded and executed at the summit.
+   This technote describes the design of an integrated night planning tool.
+   This is a tools that allows our team to collect a group of Jira tickets describing a set of tests/observations, allows us to organize these into a coherent list that can later be loaded and executed at the summit.
 
-Add content here
-================
+Introduction
+============
+
+Test planning and execution for commissioning the Vera Rubin observatory has revolved around using Jira and Confluence to generate a comprehensive plan that can be followed by the night crew.
+This process involves a combination of Jira projects:
+
+- SITCOM are used to capture high level tests/verification activities.
+  They are usually connected to one or more requirement or some other commissioning activity.
+  For example, SITCOM-1099 was created to capture the tests required to validate the operation of the system with the TMA dynamic settings going from 5% all the way to 100% of its requirements.
+
+- DM tickets are used to capture work needed either by a regular contruction deliverable or a commissioning activity.
+  DM tickets can be linked to SITCOM tickets when there is work required to accomplish a particular test, but they can also be standalone ticket that are just broadly required.
+  For instance, in the example above of SITCOM-1099, we have DM-39533 linked as a related ticket.
+  It is not strickly necessary to have this relantionship as DM-39533 represents a broader system functionality, however the issues are linked to demonstrate the relantionship.
+  Sometimes a SITCOM ticket might trigger the development of systewide functionality and the relantionship is used to indicate it.
+
+- BLOCK tickets are used to design, plan, execute and track activities at the summit.
+  Usually a BLOCK ticket will be linked to a SITCOM ticket.
+
+Up to now, planning activities at the summit have revolved around collecting the BLOCK tickets that will be executed at night and sorting them out in a table with some information about priority and order of execution.
```

Comment:
Maybe add a link to one of the night plans/logs for a practical example? Here is one for convenience:
https://confluence.lsstcorp.org/display/LSSTCOM/2024-04-23+Simonyi+Night+Log